packages/backend/services/db/dbFirebase.js

```diff
@@ -61,7 +61,11 @@ const findAllUsers = async () => {
 
 const findAllBatchedUsers = async () => {
   // get all users with a batch assigned (builderBatch prop is not null)
-  const buildersSnapshot = await database.collection("users").where("builderBatch", "!=", null).get();
+  console.log("in here!!!");
+  const buildersSnapshot = await database
+    .collection("users")
+    .where("batch.number", "!=", null || "")
```

Comment:
`.where("batch.number", ">=", 0)` might be clearer?

Reply:
Do we want batch number as a number or a string? Right now if I am not mistaken it is a string.

Reply:
I can also change it to a number. An advantage of a string could be that we could have special batch names, e.g coming from a hackathon. Then we could name it accordingly, like Batch Singapore. (just an idea)